packages/react-components/react-tags/src/components/Tag/useTagStyles.styles.ts

```diff
@@ -113,66 +60,55 @@ export const useTagBaseStyles = makeStyles({
 });
 
 const useTagStyles = makeStyles({
-  dismissableContent: {
-    paddingRight: '2px',
-  },
   dismissButton: {
     marginRight: '6px',
   },
 });
 
+const useInteractiveTagStyles = makeStyles({
+  dismissButton: {
+    ...shorthands.padding('0px', '6px'),
+    ...shorthands.borderLeft(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1),
+    borderTopLeftRadius: tokens.borderRadiusNone,
+    borderBottomLeftRadius: tokens.borderRadiusNone,
+    ...createCustomFocusIndicatorStyle({
+      borderTopLeftRadius: tokens.borderRadiusNone,
```

Comment:
you're not applying color to the focus outline?

Reply:
Not really, are you relying on outline to be the focus outline or the border?

Reply:
outline :) I joined the almighty outline church